docs/source/getting_started/movement_dataset.md

```diff
@@ -22,71 +21,148 @@ To learn more about `xarray` data structures in general, see the relevant
 
 ![](../_static/dataset_structure.png)
 
-You can always inspect the structure of a `movement` dataset `ds` by simply
-printing it:
+The structure of a `movement` dataset `ds` can be easily inspected by simply
```

Comment:
That could be neat! But also not urgent at all, I think it is very clear as it is. I will open an issue so that we can tackle when we feel like it ---> #250 

Maybe showing 2 cubes, representing a poses and a bboxes' dataset side by side?